src/app/js/formats/bubbleChart/BubbleView.js

```diff
@@ -48,7 +48,13 @@ export const BubbleView = ({ data, diameter, colorSet }) => (
                     y={y}
                     name={key}
                     value={value}
-                    color={colorSet[index % colorSet.length]}
+                    color={
+                        /(^#[0-9A-F]{6}$)|(^#[0-9A-F]{3}$)/i.test(
```

Comment:
Same code here.
Definitely, time for a function (see previous comment).

Reply:
External file `src/app/js/formats/colorUtils.js` added